### PR TITLE
May release bug fixes

### PR DESCRIPTION
**Pull request by rashmichandrashekar**
fix kube health exceptions
Add computer to ApplicationInsights Telemetry

### CODE REVIEW

File: source/code/plugin/in_kube_health.rb [hunk 1]
```diff
@@ -270,56 +271,56 @@ def process_node_condition_monitor(node_inventory)
...
-                        condition_state = HealthMonitorStates::FAIL
-                    end
-                else #Condition == READY
-                    if condition['status'].downcase != 'true'
-                        condition_state = HealthMonitorStates::FAIL
-                    end
-                end
-                details[condition['type']] = {"Reason" => condition['reason'], "Message" => condition['message'], "State" => condition_state}
+        node_inventory["items"].each do |node|
+          node_name = node["metadata"]["name"]
```
Comment (r-dilip) [inline source/code/plugin/in_kube_health.rb:275]: why this change? was this automatically done by VS code?

Comment (rashmichandrashekar) [reply source/code/plugin/in_kube_health.rb:275]: yeah its the formatting tool i have. You can ignore those. It formats every file i open.
